app/models/user.rb

```diff
@@ -291,8 +290,8 @@ def tech_admin?
   end
 
   def pro?
-    Rails.cache.fetch("user-#{id}/has_pro_membership", expires_in: 200.hours) do
-      pro_membership&.active? || has_role?(:pro)
+    Rails.cache.fetch("user-#{id}/has_pro_role", expires_in: 200.hours) do
```

Comment:
If there is no longer a pro feature, why do we even have this `def`?

Reply:
There is no longer a "Pro Membership" feature. We still have the concept of `pro` role which allows users access to analytics. Following this PR I want to rename that role `analytics` so that is makes more sense.